dbhydra/dbhydra_core.py

```diff
@@ -358,6 +361,16 @@ def connect_remotely(self):
         self.cursor = self.connection.cursor()
         print("DB connection established")
 
+    def create_new_db(self):
+        self.connection = MySQLdb.connect(host=self.DB_SERVER, user=self.DB_USERNAME,
+                                             password=self.DB_PASSWORD,
+                                             charset="utf8mb4" , cursorclass=MySQLdb.cursors.DictCursor)
+
+        self.cursor = self.connection.cursor()
+
+        create_db_command = "CREATE DATABASE " + self.DB_DATABASE
+        self.execute(create_db_command)
```

Comment:
this may be target of sql injection - https://www.w3schools.com/sql/sql_injection.asp
work with pymysql manual - https://pymysql.readthedocs.io/en/latest/user/examples.html

specifically use execute with parameters 
```
import pymysql.cursors

# Connect to the database
connection = pymysql.connect(host='localhost',
                             user='user',
                             password='passwd',
                             database='db',
                             charset='utf8mb4',
                             cursorclass=pymysql.cursors.DictCursor)

with connection:
    with connection.cursor() as cursor:
        # Create a new record
        sql = "INSERT INTO `users` (`email`, `password`) VALUES (%s, %s)"
        cursor.execute(sql, ('webmaster@python.org', 'very-secret'))

    # connection is not autocommit by default. So you must commit to save
    # your changes.
    connection.commit()

    with connection.cursor() as cursor:
        # Read a single record
        sql = "SELECT `id`, `password` FROM `users` WHERE `email`=%s"
        cursor.execute(sql, ('webmaster@python.org',))
        result = cursor.fetchone()
        print(result)
```